README.md

```diff
@@ -94,6 +94,8 @@ var ruleFinder = getRuleFinder('path/to/eslint-config')
 
 ruleFinder.getCurrentRules()
 
+ruleFinder.getCurrentRulesDetailed()
```

Comment:
Strange!! [why is this still in diff](https://github.com/sarbbottam/eslint-find-rules#as-a-required-module)?

Do you need to rebase this with `upstream/master`?